test/examples/tcpecho/setup.go

```diff
@@ -0,0 +1,58 @@
+package tcpecho
+
+import (
+	"github.com/onsi/ginkgo"
+	"github.com/rh-messaging/qdr-shipshape/pkg/testcommon"
+	"github.com/rh-messaging/shipshape/pkg/framework"
+	"github.com/rh-messaging/shipshape/pkg/framework/operators"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	Config                        *testcommon.Config
+	FrameworkNSPublic             *framework.Framework
+	FrameworkNSPrivate            *framework.Framework
+	PubCtx                        *framework.ContextData
+	PrvCtx                        *framework.ContextData
+	TcpEchoDeploymentGroupVersion = schema.GroupVersionResource{
+		Group:    "apps",
+		Version:  "v1",
+		Resource: "deployments",
+	}
+)
+
+const (
+	ENV_CLUSTER_LOCAL = "CLUSTER_LOCAL"
+)
+
+var _ = ginkgo.BeforeEach(func() {
+
+	// Loading configuration (ini or environment)
+	Config = testcommon.LoadConfig("skupper/tcpecho")
+
+	// Initializes using only Skupper Operator
+	skupperOperator := operators.SupportedOperators[operators.OperatorTypeSkupper].(*operators.SkupperOperatorBuilder)
```

Comment:
The operator that must be used in the test is the operator this PR not the master or verisoned one. Is it possible to do that?